### PR TITLE
finlocker_document_updates

### PR DESCRIPTION
**Pull request by natashareis**
Updated search events data example, saved_search and shared_search updated into saved_search_updated and saved_seach_shared, respectively

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -251,29 +248,49 @@ This evente is triggerend once the user unfavorites or unpins a favorited listin
...
+      "min_price_cents":"1",
+      "listing_image_cover_url":"",
+      "property_type":"townhouse",
+      "commute":{"mode":"car", "range":"20", "name":"work"},
+      "location":{"id":"597796fbb5e1b5f5d7000655", "name":"boston metro area", "level":"metro"
+      }
+    }
+    ]
+  },
+ "event_type":"saved_search_updated"}
```
Comment (deborasetton) [inline source/index.html.md]: This JSON is not valid.

Comment (deborasetton) [reply source/index.html.md]: Also, I'd suggest formatting this JSON the same as the other data examples.
You can use an online formatter, such as https://jsonformatter.org/json-pretty-print

File: source/index.html.md [hunk 2]
```diff
@@ -251,29 +248,49 @@ This evente is triggerend once the user unfavorites or unpins a favorited listin
...
 ```
 This event is triggered once the user, using the filter provided, performs a property search
 
-### search_saved
+### saved_search_updated
```
Comment (deborasetton) [inline source/index.html.md]: @natashareis  @AlexVKO Does this event already exist or it's going to be implemented soon?

File: source/index.html.md (hunk 1 above)
Comment (natashareis) [reply source/index.html.md]: Fixed

File: source/index.html.md (hunk 2 above)
Comment (deborasetton) [reply source/index.html.md]: Resolving as @natashareis said it's being implemented.

**Pull request merged**